self_test/faceted_surface_vtu_output/spherical_tetmesh_faceted_surface.h

```diff
@@ -0,0 +1,306 @@
+      // icosa_facet[11][0] = 1;
+      // icosa_facet[11][1] = 6;
+      // icosa_facet[11][2] = 8;
+
+      Facet_pt[12] = new TetMeshFacet(n_vertex_on_facet);
+      Facet_pt[12]->set_vertex_pt(0, Vertex_pt[1]);
+      Facet_pt[12]->set_vertex_pt(1, Vertex_pt[7]);
+      Facet_pt[12]->set_vertex_pt(2, Vertex_pt[6]);
+
+      // icosa_facet[12].resize(3);
+      // icosa_facet[12][0] = 1;
+      // icosa_facet[12][1] = 7;
+      // icosa_facet[12][2] = 6;
+
+      Facet_pt[13] = new TetMeshFacet(n_vertex_on_facet);
+      Facet_pt[13]->set_vertex_pt(0, Vertex_pt[10]);
+      Facet_pt[13]->set_vertex_pt(1, Vertex_pt[7]);
+      Facet_pt[13]->set_vertex_pt(2, Vertex_pt[1]);
+
+      // icosa_facet[13].resize(3);
+      // icosa_facet[13][0] = 10;
+      // icosa_facet[13][1] = 7;
+      // icosa_facet[13][2] = 1;
+
+      Facet_pt[14] = new TetMeshFacet(n_vertex_on_facet);
+      Facet_pt[14]->set_vertex_pt(0, Vertex_pt[10]);
+      Facet_pt[14]->set_vertex_pt(1, Vertex_pt[11]);
+      Facet_pt[14]->set_vertex_pt(2, Vertex_pt[7]);
+
+      // icosa_facet[14].resize(3);
+      // icosa_facet[14][0] = 10;
+      // icosa_facet[14][1] = 11;
+      // icosa_facet[14][2] = 7;
+
+      Facet_pt[15] = new TetMeshFacet(n_vertex_on_facet);
+      Facet_pt[15]->set_vertex_pt(0, Vertex_pt[11]);
+      Facet_pt[15]->set_vertex_pt(1, Vertex_pt[3]);
+      Facet_pt[15]->set_vertex_pt(2, Vertex_pt[7]);
+
+      // icosa_facet[15].resize(3);
+      // icosa_facet[15][0] = 11;
+      // icosa_facet[15][1] = 3;
+      // icosa_facet[15][2] = 7;
+
+      Facet_pt[16] = new TetMeshFacet(n_vertex_on_facet);
+      Facet_pt[16]->set_vertex_pt(0, Vertex_pt[7]);
+      Facet_pt[16]->set_vertex_pt(1, Vertex_pt[3]);
+      Facet_pt[16]->set_vertex_pt(2, Vertex_pt[6]);
+
+      // icosa_facet[16].resize(3);
+      // icosa_facet[16][0] = 7;
+      // icosa_facet[16][1] = 3;
+      // icosa_facet[16][2] = 6;
+
+      Facet_pt[17] = new TetMeshFacet(n_vertex_on_facet);
+      Facet_pt[17]->set_vertex_pt(0, Vertex_pt[6]);
+      Facet_pt[17]->set_vertex_pt(1, Vertex_pt[3]);
+      Facet_pt[17]->set_vertex_pt(2, Vertex_pt[9]);
+
+      // icosa_facet[17].resize(3);
+      // icosa_facet[17][0] = 6;
+      // icosa_facet[17][1] = 3;
+      // icosa_facet[17][2] = 9;
+
+      Facet_pt[18] = new TetMeshFacet(n_vertex_on_facet);
+      Facet_pt[18]->set_vertex_pt(0, Vertex_pt[9]);
+      Facet_pt[18]->set_vertex_pt(1, Vertex_pt[3]);
+      Facet_pt[18]->set_vertex_pt(2, Vertex_pt[2]);
+
+      // icosa_facet[18].resize(3);
+      // icosa_facet[18][0] = 9;
+      // icosa_facet[18][1] = 3;
+      // icosa_facet[18][2] = 2;
+
+      Facet_pt[19] = new TetMeshFacet(n_vertex_on_facet);
+      Facet_pt[19]->set_vertex_pt(0, Vertex_pt[2]);
+      Facet_pt[19]->set_vertex_pt(1, Vertex_pt[3]);
+      Facet_pt[19]->set_vertex_pt(2, Vertex_pt[11]);
+
+      // icosa_facet[19].resize(3);
+      // icosa_facet[19][0] = 2;
+      // icosa_facet[19][1] = 3;
+      // icosa_facet[19][2] = 11;
```

Comment:
If these comment blocks are not needed, please remove them.

Reply:
(This includes the other comment blocks above.)

Reply:
These have been removed in a new commit: https://github.com/oomph-lib/oomph-lib/pull/98/commits/6febfdfeb9c794f7880b4b02a0e087c25b1994d5